changedetectionio/fetch_site_status.py

```diff
@@ -1,11 +1,13 @@
-import time
-from changedetectionio import content_fetcher
-from changedetectionio import html_tools
 import hashlib
-from inscriptis import get_text
+import re
+import time
+
 import urllib3
+from inscriptis import get_text
+
```

Comment:
some minor white-lines here could be removed

Reply:
There's logic to this - there will be newlines between standard library, external libraries, and internal libraries. I did notice that there was a duplicate import of `html_tools` though, which I'll remove.

Reply:
Super cool! I'm learning a lot from you :dancers: